modules/docs/mdx/10.0-UPGRADE-GUIDE.mdx

```diff
@@ -128,23 +117,24 @@ these sizes should change along with it. Since `px` are a fixed unit and do not
 
 ## Component Updates
 
-### Buttons
-
-We re-factored how we styled Buttons to use our `createStyles` utility function. We don't anticipate
-this as a breaking change but, there may be slight changes to visual test.
+### Button
 
 **PR:** [#2285](https://github.com/Workday/canvas-kit/pull/2285)
 
+We've refactored how we style buttons to use our `createStyles` utility function. We don't
+anticipate this being a breaking change in most cases, but there may be slight changes to visual
+tests.
+
 #### Button Icon Fill
 
-Icons will no longer be "filled" on toggle. This decision was made to not have the exisitng icon
-look so different in the toggled state.
+In an effort to soften the visual effect when a button is toggled, icons will no longer be "filled"
+on toggle.
 
 #### Button Focus Ring Update
 
-We found that focus ring was not consistent across all Buttons. We have made some changes to create
-a consistent focus ring across all Buttons and variants. These will need to be taken note of due to
-some small changes visually with Buttons.
+We found that focus rings were not consistent across all buttons. We've updated our buttons to
+display consistent focus rings across all variants. These will need to be taken note of due to some
+small changes visually with Buttons.
 
 - `PrimaryButton`: `inverse` variant now has the same focus ring as default variant and as
   `SecondaryButton`.
```

Comment:
Is the "**These**" in "**These** will need to be taken note of due to some small changes visually with Buttons" referring to the single item for `PrimaryButton` listed below? No need for this to be list since there's only a single item now (I do recall `TertiaryButton` being in the list at one point).

Reply:
`TertiaryButton` was removed as design did not want to change the focus ring. 
`PrimaryButton` was changed, that's just the inverse variant that now matches others